cli/Cargo.toml

```diff
@@ -63,7 +64,8 @@ experimental = [
 ]
 
 circuit = ["reqwest", "serde_json", "splinter/sawtooth-signing-compat", "dirs", "uuid"]
-circuit-auth-type = []
+circuit-template = ["splinter/circuit-template", "circuit"]
+circuit-auth-type = ["circuit"]
```

Comment:
I can't tell why this feature is changed to require the `circuit` feature; can you explain this in the commit message?

Reply:
Moved that change to it's own commit. The `circuit-auth-type` feature is part of the `circuit` feature. It does not make sense enabling the `circuit-auth-type` feature 
if the `circuit` feature is not enabled.